tools/check.env

```
@@ -0,0 +1,11 @@
+# Do not report if package size is larger than 5 megabytes
+_R_CHECK_PKG_SIZES_=false
+
+# Do not check Rd cross references
+_R_CHECK_RD_XREFS_=false
+
+# Do not report if package requires GNU make
+_R_CHECK_CRAN_INCOMING_NOTE_GNU_MAKE_=false
```

```suggestion
_R_CHECK_CRAN_INCOMING_NOTE_GNU_MAKE_=true
```

My understanding of the code here seems to suggest it should be `true`. That's somewhat counter intuitive but can we give it a try please?

https://github.com/wch/r-source/blob/b12ffba7584825d6b11bba8b7dbad084a74c1c20/src/library/tools/R/check.R#L2960C2-L2960C2